.gitignore

```diff
@@ -1,3 +1,5 @@
+# Cython extensions
+var_int.c
```

Comment:
The Cython documentation says [here](https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#distributing-cython-modules)

> It is strongly recommended that you distribute the generated .c files as well as your Cython sources, so that users can install your module without needing to have Cython available.

Reply:
Oh, nice, I didn't know that